library/GN.md

```diff
@@ -9,10 +10,32 @@ become the build system used on all platforms.
 
 In addition to the normal dependencies, you will need to install:
 * [ninja](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages)
-* [gn](https://gn.googlesource.com/gn/) 
+* [gn](https://gn.googlesource.com/gn/)
 
 Ensure that both binaries are in your path.
 
+Windows also requires the 64 bit compiler, linker and setup scripts to be in
+your path. They are found under:
+
+```
+> Visual Studio Install Path\2017\Version\VC\Auxiliary\Build
+```
+
+e.g.:
+
+```
+> C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build
+```
+
+Windows requires jsoncpp to be downloaded to
+`library/windows/third_party/jsoncpp`. Use
+`tools/dart_tools/bin/fetch_jsoncpp.dart` to automatically download `jsoncpp`
+with Visual Studio 2017 support as shown below:
+
+```
+> tools\run_dart_tool.bat tools\dart_tools\bin\fetch_jsoncpp.dart library\windows\third_party\jsoncpp
+```
```

Comment:
To reduce the changes of someone having issues without any clue what the problem is, I think it would be good to have a warning about the fact that the jsoncpp GN build is currently a placeholder, and if they update or modify the jsoncpp tree they should delete `out/gen/JSON` (not on Windows at the moment, so I may have that path wrong) then rebuild.